app/models/budget.rb

```diff
@@ -152,13 +152,13 @@ def investments_orders
   end
 
   def email_selected
-    investments.selected.each do |investment|
+    investments.selected.order(:id).each do |investment|
```

Comment:
The commit message says:

> Ordering by id is certainly not very elegant; however, ordering by
another field like `created_at` is dangerous because the record could be
created at (almost) the exact same time.

However, `created_at` stores the time up to microseconds, so the chance of two records being created at the exact same time is zero.